MSProject/LibraryCollectionView/WBLibraryCollectionViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  WBLibraryCollectionViewController.swift
+//  MSProject
+//
+//  Created by Matias Spinelli on 11/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class WBLibraryCollectionViewController: UIViewController {
+
+    private let libraryCollectionView: WBLibraryCollectionView = WBLibraryCollectionView.loadFromNib()!
+    var libraryItems: [WBBook] = []
+    
+    override func loadView() {
+        view = libraryCollectionView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        configureCollectionView()
+        
+        loadBooks()
+        
+        title = "LIBRARY".localized() + "Collection"
+    }
+    
+    // MARK: - Private
+    private func configureCollectionView() {
+        libraryCollectionView.libraryCollectionView.delegate = self
+        libraryCollectionView.libraryCollectionView.dataSource = self
+        
+        libraryCollectionView.libraryCollectionView.backgroundColor = UIColor.woloxBackgroundLightColor()
```

Comment:
Two little things about this:

1. The extension for the app colors is a good idea! I would personally make those colors computed properties, so that they end up looking exactly like the default colors (`UIColor.black`, for example).

2. You could take advantage of type inference here and just do (if you end up changing the other properties):
`_view.bookCollection.backgroundColor = .woloxBackgroundLightColor`

Yup, I'm all about syntax sugar.